subworkflows/nf-core/utils_nextflow_pipeline/meta.yml

```diff
@@ -1,5 +1,5 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
-name: "nextflow_pipeline_utils"
+name: "UTILS_NEXTFLOW_PIPELINE"
```

Comment:
the PIPELINE ending gives the feeling this is it's own util pipeline. But i also see that naming was already much discussed. so I"ll stay out of it :D